### ISSUE TITLE
Encoding in decompressed data needs to be handled to be ascii only

### ISSUE THREAD

**Issue by brokensound77** [brokensound77/AlertLogic-event-api#12]
return from `gz_handler`

possible solution below

```python
returned_compressed_data.decode('ascii', 'ignore'))
new_returned_compressed_data = ''.join([c for c in returned_compressed_data if c in printable])
```

**Comment by brokensound77** [comment]
This was addressed in 41eda76689c7761a92b099cb7ce4d72168106ad5 in the master-dev branch

After sustained testing, if there are no issues, I will pull to the master branch

cc: @TheRemover

**Comment by brokensound77** [comment]
Updated and merged in #16
